index.html

```diff
@@ -39,19 +42,27 @@ <h4 class="connectmywork_01"> LET’S CONNECT</h4>
     </div>
 </section>
 <section class="gridcontainer">
```

Comment:
Please make sure you add enough margin or padding to the right and left of the card according to the Figma design. 
Figma template:
<img width="803" alt="Screenshot 2022-12-16 at 9 58 42 PM" src="https://user-images.githubusercontent.com/25887253/208174659-67c9c7ca-bfbf-4865-85f9-c246120f84f6.png">

Your design:
<img width="1440" alt="Screenshot 2022-12-16 at 9 59 09 PM" src="https://user-images.githubusercontent.com/25887253/208174699-0fc25462-3b7c-4f92-a4df-79d612804c39.png">